src/supermin-init-prelude.sh

```diff
@@ -47,7 +47,6 @@ for maybe_symlink in "${workdir}"/{src/config,src/yumrepos}; do
     fi
 done
 
-mkdir -p "${workdir}"/cache
```

Comment:
sometimes I just `rm -rf cache/` to blow away things to get a clean slate. I'm guessing something else would create it somewhere else in the stack, but...

could we just move this into the if statement below? if there is a cachedev and somehow the cache dir doesn't exist we'd surely want to create it, right?

Reply:
Hmm, but the `cache/` dir must already exist at that point if using a cache qcow2, since the qcow2 is located there, right?

Reply:
Hmm, I feel like whatever/whoever does `rm -rf cache/` should ensure to recreate it. E.g. rpm-ostree today will error out trying to cache things there. But not against adding a `mkdir -p cache` somewhere in `cmdlib.sh` too.